### ISSUE TITLE
Sorting by multiple fields

### ISSUE THREAD

**Issue by nojacko** [holidayextras/jsonapi-server#338]
Currently sorting is only available on one attribute. I was looking to add sorting by multiple attributes. 

I was planning on doing it using CSV, similar to includes and filters. 

i.e. `sort=-attr_one,+attr_two,-attr_three`

Is this something you'll be cool with if I get it working and make a PR?

**Comment by paparomeo** [comment]
Yes, that would definitely be a welcome contribution. Have a look at what the [spec](http://jsonapi.org/format/#fetching-sorting) says about sorting. It's pretty much your proposed format but the `+` is implicit instead of explicit.
